purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -126,6 +131,29 @@ fun Purchases.purchaseProductWith(
     purchaseProduct(activity, storeProduct, purchaseCompletedCallback(onSuccess, onError))
 }
 
+/**
+ * Purchase product.
+ * @param [activity] Current activity
+ * @param [storeProduct] The storeProduct of the product you wish to purchase
+ * @param [purchaseOption] Your choice of purchase options available for the StoreProduct
+ * @param [onSuccess] Will be called after the purchase has completed
+ * @param [onError] Will be called after the purchase has completed with error
+ */
+fun Purchases.purchaseProductWithOption(
```

Comment:
The naming of these functions is the same as the main ones in `Purchases.kt`, but since the signature is different there is no collision. Lmk if that's a concern...